core-redirects-studio-lib/src/main/java/com/tallence/core/redirects/studio/rest/RedirectResource.java

```diff
@@ -72,8 +87,17 @@ private String getId() {
     return id;
   }
 
+  private String getSiteId() {
+    return siteId;
+  }
+
+  @PathParam("siteId")
+  public void setSiteId(String siteId) {
```

Comment:
Die Site hier über einen Setter reinzureichen statt die direkt als Methoden-Parameter kommt mir komisch vor. Ist das Thread-safe?

Reply:
Ich denke es ist kein Problem. Coremedia verwendet diesen Mechanismus in seinen Resource-Klassen ebenfalls. Beispiel: com.coremedia.collaboration.project.rest.ProjectResource

Reply:
Ok, in der Doku zu `@PathParam` steht:
> Because injection occurs at object creation time, ....
Was effektiv bedeutet, dass für jeden Aufruf eine eigene Instanz erzeugt wird. Das wusste ich nicht...